### DISCUSSION TITLE
Can we use json output from a shell script?

### DISCUSSION THREAD

**Discussion by Animeshz** [elkowar/eww#316] [Q&A] [answered]
Alot of inbuilt properties (magic vars) look like JSON in the `eww state` output, can we use json from a shell script output in `defpoll|deflisten`?

**Comment by elkowar** [comment, marked as answer]
Yes! The simplexpr expression language embedded into yuck supports json access, as listed [here in the docs](https://elkowar.github.io/eww/expression_language.html#features). As long as the string is valid json, you can treat it as such
